airbyte_ui_rock/rockcraft.yaml

```diff
@@ -0,0 +1,75 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: airbyte-ui
+summary: Airbyte UI rock
+description: Airbyte UI OCI image for the Airbyte UI charm
+version: "1.0"
+base: ubuntu@22.04
+build-base: ubuntu@22.04
+license: Apache-2.0
+platforms:
+  amd64:
+
+services:
+  airbyte-webapp:
+    override: replace
+    summary: "airbyte-webapp service"
+    startup: disabled
+    command: "/usr/bin/pnpm -C airbyte-webapp start"
+    environment:
+      PORT: "8080"
+
+parts:
+  patches:
+    plugin: dump
+    source: ./patches
+    organize:
+      vite-config-http.patch: patches/vite-config-http.patch
+    stage:
+      - patches/vite-config-http.patch
+    prime:
+      - "-*"
+
+  airbyte-webapp:
+    after: [patches]
+    plugin: dump
+    source: https://github.com/airbytehq/airbyte-platform.git # yamllint disable-line
+    source-type: git
+    source-commit: 34d28a9a8d3092a7e22eec6562946e6ef6ca36ab # v0.60.0
```

Comment:
Suggestion: if there is a git tag that you can use this might be better as I've had commits I'd pinned force changed before.

Reply:
TIL, thanks for the suggestion!